src/applause-button.test.js

```diff
@@ -111,7 +113,7 @@ describe("Applause button", () => {
     it("send the updated clap count to the server", async () => {
       await createPage();
       await clickTheButton();
-      await pause(2500);
+      await pause(2200);
```

Comment:
For my understanding, why do we need this change?

Reply:
Hah yes, good question! The test runs were way too close to timing out for comfort, so I reduced these pauses a little; in a subsequent commit (PR to follow) I tackle this in a better way by increasing the overall test timeout.

I did attempt to `useFakeTimers` as we are using Jest under the bonnet, however, that did not appear to work with the jest-puppeteer preset, so it seems we are stuck with _real_ timeouts, hence why these tests take a while to run.

A better solution might be to run most of our tests using Jest and Testing Library, and simply run a couple of smoke tests using puppeteer. Or, we could switch to [Playwright and its Clock API](https://playwright.dev/docs/clock).